docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -24,13 +24,38 @@ embedded assets. This can be different for older releases.
 
 ## Download
 
-Choose the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/), copy the URL and replace the URL within the commands below:
+You can find the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/) after navigating to the version you want to download. 
+
+### Choosing the right file
+For Linux, you should choose: 
+ * `linux-386` for 32-bit Intel/AMD processors
+ * `linux-amd64` for 64-bit Intel/AMD processors
+ * 'linux-arm64' for 64-bit ARM systems like a Raspberry PI 4
+ * `linux-arm-5` and `linux-arm-6` are for other versions of the ARM processor 
+ 
+For Windows, you should chose:
+ * `gogit-windows-4.0-386` for 32-bit Intel/AMD processors
```

Comment:
No. 

You should consider whether the gogit variant will work better for you or not. 

It's not guaranteed otherwise we'd only provide one or the other.

and again the OS is the discriminator as to whether you should run 386 or amd64 versions.